File: ToyLanguageInterpretor/src/model/file/FileData.java
package model.file;
import java.io.BufferedReader;

public class FileData {
	private String FName;
	private BufferedReader buffer;
	
	public FileData(String name,BufferedReader buff){
		FName = name;
		buffer=buff;
	}
	
	public String toString(){
		return FName;
	}

	public BufferedReader getReader() {
		return buffer;
	}
	public String getFileName(){
		return FName;
	}
}

File: ToyLanguageInterpretorC#/ToyLanguageC/model/FileData.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ToyLanguageC.model
{
    class FileData
    {
        private string name;
        private StreamReader buffer;

        public FileData(string name,StreamReader buffer)
        {
            this.name = name;
            this.buffer = buffer;
        }
        public string Name
        {
            get { return name; }
            set { name = value; }
        }

        public StreamReader Buffer
        {
            get { return buffer; }
            set { buffer = value; }
        }

        public string toString()
        {
            return name;
        }

    }
}
